IRC-Relay/Helpers.cs

```diff
@@ -35,58 +34,23 @@ public static string UploadMarkDown(string input)
 
         public static string MentionToUsername(string input, SocketUserMessage message)
         {
-            string returnString = message.Content;
-
-            Regex regex = new Regex("<@!?[0-9]+>");
-            Match match = regex.Match(input);
-            if (match.Success) // contains a mention
-            {
-                string substring = input.Substring(match.Index, match.Length);
-
-                SocketUser user = message.MentionedUsers.First();
-
-
-                returnString = input.Replace(substring, user.Username);
-            }
+            return Regex.Replace(input, "<@!?([0-9]+)>", m => message.MentionedUsers.First(u => u.Id.ToString() == m.Value).Username);
```

Comment:
This check edit - along with what was done to `ChanelMentionToName()` - causes nothing to be relayed to IRC. Consider removing the user check, as I believe that's what's causing this to fail. Replacing it with
```csharp
m => message.MentionedUsers.First().Username
```
will likely fix this issue. Similar things would need to be done for `ChanelMentionToName()`